src/test/ui/const-generics/const_evaluatable_checked/from-sig-fail.stderr

```diff
@@ -1,9 +1,20 @@
-error[E0080]: evaluation of constant value failed
-  --> $DIR/from-sig-fail.rs:4:35
+error: failed to evaluate the given constant
+  --> $DIR/from-sig-fail.rs:9:5
    |
 LL | fn test<const N: usize>() -> [u8; N - 1] {
-   |                                   ^^^^^ attempt to compute `0_usize - 1_usize`, which would overflow
+   |                                   ----- required by this bound in `test`
```

Comment:
This is an error quality regression.

Reply:
I don't think so. We previously didn't even mention that the errors was caused by the `test::<0>()` call which is definitely an improvement.

We probably want to reevaluate the constant when emitting the selection error to actually emit why the evaluation failed. at which cause we get both error messages.